File: PnSQL/web/src/api/sys_user.js
import request from "@/request/index.js"

export const GetSysUser = (id)=>{
    return request.get('/v1/sys/user/post/$(id)')
}

export const AddSysUser = (user)=>{
    return request.post('/v1/sys/user/addUser',user)
}

export const UptSysUser = (row)=>{
    return request.post('/v1/sys/user/uptUser',row)
}
export const UptSysUserFiled = (params)=>{
    return request.post('/v1/sys/user/uptFiled',params)
}

export const DelSysUser = (id)=>{
    return request.post(`/v1/sys/user/delUser?id=${id}`)
}

export const LoadSysUserData = (params)=>{
    return request.post("/v1/sys/user/load",params)
}

File: router/example/sys_user.go
package example

import (
	"github.com/gin-gonic/gin"
	v1 "github.com/xiaohongshu/PnSql/server/api/v1"
)

type SysUserRouter struct{}

func (s *SysUserRouter) InitSysUserRouter(Router *gin.RouterGroup) {
	sysUserRouter := Router.Group("/sys")
	sysUserApi := v1.ApiGroupApp.ExampleApiGroup.SysUserApi
	{
		sysUserRouter.GET("/user/post/:id", sysUserApi.GetById)
		sysUserRouter.POST("/user/addUser", sysUserApi.AddSysUser)
		sysUserRouter.POST("/user/uptUser", sysUserApi.UptSysUser)
		sysUserRouter.POST("/user/uptFiled", sysUserApi.UptSysUserField)
		sysUserRouter.POST("/user/delUser", sysUserApi.DelSysUser)
		sysUserRouter.POST("/user/load", sysUserApi.LoadSysUserPage)
	}
}
